### PR TITLE
Create python-publish.yml

### PR DESCRIPTION
**Pull request by duongxuantuan154**

### CODE REVIEW

**Comment by duongxuantuan154** [comment]
Duplicate of #

**Review by duongxuantuan154 — COMMENTED** [review]
> ``

File: .github/workflows/python-publish.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  deploy:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.x'
```
Comment (hoffmang9) [inline .github/workflows/python-publish.yml:25]: You will need to specify the version as we do not support anything less than python 3.7.

**Review by hoffmang9 — CHANGES_REQUESTED** [review]
Can you tell us more about your motivation? We already upload a pure python source distribution to Pypi.

**Review by duongxuantuan154 — COMMENTED** [review]
Đóng

File: .github/workflows/python-publish.yml (hunk 1 above)
Comment (duongxuantuan154) [reply .github/workflows/python-publish.yml:25]:
```suggestion
        python-version: '3.x'
```

Comment (duongxuantuan154) [reply .github/workflows/python-publish.yml:25]:
```suggestion
        python-version: '3.7.'
```

**Review by duongxuantuan154 — COMMENTED** [review]
#7104

**Review by duongxuantuan154 — COMMENTED** [review]
@duongxuantuan154

File: .github/workflows/python-publish.yml (hunk 1 above)
Comment (duongxuantuan154) [reply .github/workflows/python-publish.yml]: https://github.com/Chia-Network/chia-blockchain/pull/7105/files/8758dc442a073b59417c1407c3d4db4c2fb4e1b7#r663290462

**Review by duongxuantuan154 — COMMENTED** [review]
- [ ]

**Review by duongxuantuan154 — COMMENTED** [review]
- [ ]

**Review by duongxuantuan154 — COMMENTED** [review]
- [ ]

File: .github/workflows/python-publish.yml (hunk 1 above)
Comment (duongxuantuan154) [reply .github/workflows/python-publish.yml]: 3.0.7

**Review by duongxuantuan154 — COMMENTED** [review]
- [ ]

**Pull request closed without merging**